LDAR_Sim/src/time_counter.py

```diff
@@ -56,7 +56,11 @@ def set_UTC_offset(self, sites):
         '''
         avg_lat = mean([float(site['lat']) for site in sites])
         avg_lon = mean([float(site['lon']) for site in sites])
-        timezone_str = tzwhere.tzwhere().tzNameAt(avg_lat, avg_lon)
-        # This uses the current daylight time . Need to fix
+        tf = TimezoneFinder()
+        timezone_str = tf.timezone_at(lng=avg_lon, lat=avg_lat)
+        # This uses the current time to estimate offset, so if running
+        # software during DST, the the offset will include DST. Fix this
+        # someday, by keeping timezone as a site variable and localizing
+        # very year.
```

Comment:
UTC offsets are very complicated - I'd argue for depreciating local time from this model eventually! Daylight savings schemes always are changing and have wacky jurisdictional boundaries.

Reply:
Interesting ! We dont really have a choice because of hourly weather which is in UTC. That timezonefinder / pytz libraries consider all locations, and actually have the DST rules built in. Although yeah these are always changings so we may have to keep that library up to date as best as we can. Right now, regardless of where you are, the local time calculated is correct to within an hour, which considering we were using 24 hour averages is acceptable. 

More thought on this would be awesome.